presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -258,6 +259,19 @@ public int getMaxQueryRunningTaskCount()
         return maxQueryRunningTaskCount;
     }
 
+    @Config("experimental.max-total-running-task-count-to-halt-scheduling")
```

Comment:
nit: `scheduling` usually means task scheduling or stage scheduling in Presto. What about 
```
experimental.max-total-running-task-count-for-new-query-execution
```

Reply:
> experimental.max-total-running-task-count-for-new-query-execution

Changed to ``experimental.max-total-running-task-count-to-not-execute-new-query``